src/components/Navbar/styles.scss

```diff
@@ -85,36 +85,17 @@
       padding-left: 20px;
     }
 
-    &--children-selected {
-      background-color: transparent;
```

Comment:
If we remove this, then the "selected parent" will be colored also. Is that intentionally?

Reply:
Yes, I got that from the spec.
<img width="118" alt="Navigation 2023-01-13 06-18-03" src="https://user-images.githubusercontent.com/1865174/212341183-e658aeeb-d0d2-49e5-be55-5e7af01640dc.png">

I don't know if this is correct, will let @hannah-cai comment?

Reply:
hmmm. my first instinct is that we shouldn't, but then if users close the dropdown they wouldn't have a way to find which page they're on. I thought that we weren't letting users close dropdowns that they're currently in? currently you can close it.

cc @chudo76 for his thoughts too

Reply:
Yes - you can close it now.

Reply:
@karthikbtf: @chudo76 and I discussed briefly and we want to keep the nav the way it is currently, where if a dropdown is active you can't close it, and where the highlight is full width even if nav item is a child of a parent. 

on an unrelated note, I'm not sure if this is a bug but the third dropdown's items seem to be incorrect? I'm seeing this happening
<img width="276" alt="image" src="https://user-images.githubusercontent.com/45954862/212438025-8bfe7569-5b56-4259-b548-56da8db41a74.png">

Reply:
the bug is in the test-case, i will clean that up.. and make the other changes.

Reply:
@karthikbtf sorry! I'd forgotten to mention this in my earlier comment, but Chudo and I also thought that we should remove the coloring of the parent (in other words, keep this code instead of removing it). if you can't close active dropdowns then users losing their place on the menu is no longer a concern, so we don't need to have the parent highlighted. it also gets rid of the double highlighted issue you pointed out below, and shows more clearly at a glance which tab a user is on

Reply:
sounds good 👍

Reply:
@hannah-cai do we want the hover state though?

Reply:
updated - removed coloring of the parent.

Reply:
hmm, good question. is there a way you can turn it off if it's active, but keep it if it isn't?

Reply:
Yes, possible, updated with those changes. Please take a look in vercel.